_pages/en/it-picture-medium-term.md

```diff
@@ -6,11 +6,34 @@ In the following sections, we're trying to take the same end state and create "v
 
 ## Citizen
 
+- Mix of old and new..
+- Current portals and accounts
+- OneGC
+- API Store
+- Pan-Canadian Trust Framework
+- Some benefits integrated with 3rd parties
+
 ## Businesses
 
+- Able to find and integrate with 50% of ESDC services
+
 ## Non-profits
 
-## IITB
+- Able to find and integrate with 50% of ESDC services
+
```

Comment:
Should we use some of things identified in KPIs in relation to the operational efficiency and delivery of IT services/products?

Reply:
Aye, I think we should ... though we are intending to fiddle with the KPI document a fair bit in the near future ...